actioncable/lib/action_cable/connection/base.rb

```diff
@@ -52,47 +51,37 @@ class Base
       include Callbacks
       include ActiveSupport::Rescuable
 
-      attr_reader :server, :env, :subscriptions, :logger, :worker_pool, :protocol
-      delegate :event_loop, :pubsub, :config, to: :server
+      attr_reader :subscriptions, :logger
+      private attr_reader :server, :raw_conn
```

Comment:
What is `:raw_conn`? Can we make the name a little more explicit?

Reply:
Ended up with "socket"